docs/update-user-guide-links.py

```diff
@@ -0,0 +1,85 @@
+import subprocess
+import re
+
```

Comment:
Since this isn't run w/ every build, could we put it in a _tools or _scripts directory?